e2e/tests/components/input/with-error.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, test } from '../../../fixtures'
+
+test.describe('input, with error', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/components/input/with-error')
+  })
+
+  test('displays the heading', async ({ page }) => {
+    await expect(
+      page.getByRole('heading'),
+    ).toHaveText('What is the name of the event?')
+  })
+
+  test('displays the input', async ({ page }) => {
+    await expect(page.getByRole('textbox')).toBeVisible()
+  })
+
+  test('displays the error message', async ({ page }) => {
+    await expect(page.getByRole('paragraph')).toHaveText('Error: Enter an event name')
+  })
+})
```

Comment:
I'm not to happy with this spec I wanted to use a locator similar to this https://testing-library.com/docs/queries/byrole/#description

Reply:
It probably wouldn't be much effort to add a custom `toHaveDescription` assertion to `playwright.config.ts` and `global.d.ts` also